src/main/java/com/stripe/model/Plan.java

```diff
@@ -193,8 +193,18 @@ public Plan update(Map<String, Object> params, RequestOptions options) throws St
   @Setter
   @EqualsAndHashCode(callSuper = false)
   public static class Tier extends StripeObject {
-    Long amount;
+    Long unitAmount;
     Long upTo;
+
+    /**
+     * The {@code amount} attribute.
+     *
+     * @return the {@code amount} attribute
+     * @deprecated Prefer using the {@code unitAmount} attribute instead.
+     * @see <a href="https://stripe.com/docs/upgrades#2018-08-23">API version 2018-08-23</a>
+     */
+    @Deprecated
```

Comment:
Is this attribute really deprecated? It's still present in the API reference: https://stripe.com/docs/api#plan_object-amount

Reply:
It is actively deprecated since we shipped `unit_amount`. We're renaming amount because we are going to ship another type of amount property and we wanted to to avoid the confusion.

TL;DR: yep, the docs are incorrect I suppose